### ISSUE TITLE
New Smartthings app/platform

### ISSUE THREAD

**Issue by pittbull** [stjohnjohnson/smartthings-mqtt-bridge#219]
Hi. 

Do you, or anybody else, know if the bridge will continue to work if we migrate to the new app?

**Comment by dlasher** [comment]
Migrated everything yesterday, was worried about all my DTH's, discovered today MQTT stopped working. So, first blink, no?

**Comment by jfgibbins** [comment]
samsung hosed everything with the new improvements that don't do anything.  Nice job samsung!!

**Comment by admin4fun** [comment]
This is correct, wish I had noticed this before uninstalling my openhab2 smartthing binding that was working and installing docker, setting it up -.-

My dumb fault. I can confirm docker is running port 8080 is open, no traffic is received from smartthing hub via tcpdump. Connection from docker to mosquitto seems to work, no data.

**Comment by jmcclure0921** [comment]
I have the new app and it all works correctly.

**Comment by orenatan** [comment]
Any solution?

**Comment by Artekus** [comment]
Mine has worked with out issue. I updated my DH a couple months ago to allow more info from my refrigerator to be passed and it's still working fine

**Comment by robross0606** [comment]
I appear to be getting all sorts of timeouts lately.  I cannot even delete my "MQTT Bridge" device or the SmartApp because I get this error:

> Hibernate operation: could not delete: [physicalgraph.device.DevicePreferences#d2404290-d3ed-402c-bdbb-5092d665bb7d]; uncategorized SQLException for SQL [delete from device_preferences where id=? and version=?]; SQL state [25S03]; error code [0]; (conn=1125550) Communications link failure with primary host na02useast1-datamgmt.cluster-cojnp12rqtju.us-east-1.rds.amazonaws.com:3310. Connection timed out; nested exception is java.sql.SQLException: (conn=1125550) Communications link failure with primary host na02useast1-datamgmt.cluster-cojnp12rqtju.us-east-1.rds.amazonaws.com:3310. Connection timed out

**Comment by bogus83** [comment]
I am getting similar errors when trying to configure the SmartApp for the first time.  Is this functional or should I abandon further efforts?

10:02:02 PM: debug Installed with settings: [battery:[Bedroom Sensor, Office Sensor], bridge:MQTT Bridge]

10:02:22 PM: error org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: Could not execute JDBC batch update; uncategorized SQLException for SQL [insert into event_subscription (version, data, date_created, device_id, filter_events, handler, hub_id, installed_smart_app_id, last_updated, location_id, name, id) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]; SQL state [25S03]; error code [0]; (conn=27053913) **Communications link failure with primary host na04useast2-datamgmt.cluster-cwcarkwvvr9r.us-east-2.rds.amazonaws.com:3310. Connection timed out; nested exception is java.sql.BatchUpdateException: (conn=27053913) Communications link failure with primary host na04useast2-datamgmt.cluster-cwcarkwvvr9r.us-east-2.rds.amazonaws.com:3310. Connection timed out @line 504 (initialize)**
